modular_skyrat/modules/cortical_borer/code/procs.dm

```diff
@@ -0,0 +1,5 @@
+/mob/living/carbon/human/proc/hurt_organ_randomly(damage) // Gamble on your organs...
+	for(var/i in internal_organs)
+		if(prob(100 / length(internal_organs)))
```

Comment:
why are you using prob here? use rand() or something.

Reply:
i failed algebra yet i try